frontend/pages/sign-up.tsx

```diff
@@ -0,0 +1,299 @@
+                className="mt-1 ml-2 font-sans text-xs text-red font-regular"
+              >
+                {errors.firstName.message}
+              </p>
+            )}
+          </div>
+          <div className="w-full">
+            <label htmlFor="last-name">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Last name" id="txUL0F" />
+              </p>
+              <Input
+                type="text"
+                name="lastName"
+                id="last-name"
+                placeholder="Insert your last name"
+                aria-describedby="last-name-error"
+                register={register}
+                registerOptions={{
+                  required: intl.formatMessage({
+                    defaultMessage: 'You need to enter a last name.',
+                    id: 'zPMPr9',
+                  }),
+                }}
+              />
+            </label>
+            {errors.lastName && (
+              <p id="last-name-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+                {errors.lastName.message}
+              </p>
+            )}
+          </div>
+        </div>
+        <div className="w-full">
+          <label htmlFor="email">
+            <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+              <FormattedMessage defaultMessage="Email" id="sy+pv5" />
+            </p>
+            <Input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Insert your email"
+              aria-describedby="email-error"
+              register={register}
+              registerOptions={{
+                required: intl.formatMessage({
+                  defaultMessage: 'You need to enter your email',
+                  id: 'NioTuC',
+                }),
+                validate: (value) =>
+                  validate(
+                    { from: value },
+                    {
+                      from: {
+                        email: true,
+                      },
+                    }
+                  ),
+              }}
+            />
+          </label>
+          {errors.email && (
+            <p id="email-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+              {errors.email.message || (
+                <FormattedMessage defaultMessage="Invalid email format" id="nc1IrM" />
+              )}
+            </p>
+          )}
+        </div>
+        <div className="flex gap-4">
+          <div className="w-full">
+            <label htmlFor="password">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Password" id="5sg7KC" />
+              </p>
+              <Input
+                type="password"
+                placeholder="insert password"
```

Comment:
Needs to be translated. For consistency with the other placeholder, you should also capitalise: `"Insert password"`.